ister.py

```diff
@@ -1422,6 +1478,21 @@ def install_os(args, template):
         copy_os(args, template, target_dir)
         add_users(template, target_dir)
         set_hostname(template, target_dir)
+        # UI changes override command line
+        if args.contenturl or template.get("MirrorURL"):
```

Comment:
I think that we shouldn't persist the args to disk, only the template in this case.

Reply:
@mdhorn - isn't it true that the new installer will persist the command line args to disk on target host?   My intent here is to be consistent with what new installer will do (physical servers only)